### ISSUE TITLE
Add PL 0.9 Features

### ISSUE THREAD

**Issue by dthiagarajan** [dthiagarajan/ptb-xl-research#14]
Include PL 0.8.1 features and 0.9 features, as this is the final API:

* (0.8.1 features)[https://medium.com/pytorch/pytorch-multi-gpu-metrics-and-more-in-pytorch-lightning-0-8-1-b7cadd04893e]
* (0.9 features)[https://medium.com/pytorch/pytorch-lightning-0-9-synced-batchnorm-datamodules-and-final-api-aae885398a9d]
* (Ray tune potential)[https://anyscale.com/blog/scaling-up-pytorch-lightning-hyperparameter-tuning-with-ray-tune/]
